docs/designers-developers/glossary.md

```diff
@@ -1,21 +1,61 @@
+
+<dt>Block</dt>
+<dd>The abstract term used to describe units of markup that, composed together, form the content or layout of a webpage. The idea combines concepts of what in WordPress today we achieve with shortcodes, custom HTML, and embed discovery into a single consistent API and user experience.</dd>
+
+<dt>Block Categories</dt>
+<dd>These are not a WordPress taxonomy, but instead used internally to sort blocks in the Block Inserter.</dd>
+
+<dt>Block Inserter</dt>
+<dd>Primary interface for selecting from the available blocks, triggered by plus icon buttons on Blocks or in the top-left of the editor interface.</dd>
+
+<dt>Block name</dt>
+<dd>A unique identifier for a block type, consisting of a plugin-specific namespace and a short label describing the block's intent. e.g. `core/image`</dd>
+
+<dt>Block type</dt>
+<dd>In contrast with the blocks composing a particular post, a block type describes the blueprint by which any block of that type should behave. So while there may be many images within a post, each behaves consistent with a unified image block type definition.</dd>
+
+<dt>Classic block</dt>
+<dd>A block which embeds the TinyMCE editor as a block, TinyMCE was the base of the previous core editor. Older content created prior to the block editor will be loaded in to a single Classic block.</dd>
+
+<dt>Dynamic block</dt>
+<dd>A type of block where the content of which may change and cannot be determined at the time of saving a post, instead calculated any time the post is shown on the front of a site. These blocks may save fallback content or no content at all in their JavaScript implementation, instead deferring to a PHP block implementation for runtime rendering.</dd>
+
+<dt>Inspector</dt>
+<dd>A block settings region shown in place of the post settings when a block is selected. Fields may be shown here to allow the user to customize the selected block.</dd>
+
+<dt>Post settings</dt>
+<dd>A sidebar region containing metadata fields for the post, including scheduling, visibility, terms, and featured image.</dd>
+
+<dt>RichText</dt>
+<dd>A common component enabling rich content editing including bold, italics, hyperlinks, etc. It is not too much unlike the single editor region of the legacy post editor, and is in fact powered by the same TinyMCE library.</dd>
+
+<dt>Reusable block</dt>
+<dd>A block that is saved and then can be shared as a reusable, repeatable piece of content.</dd>
+
+<dt>Sidebar</dt>
+<dd>The panel on the right which contains the document and block settings. The sidebar is toggled using the Settings gear icon.</dd>
+
+<dt>Serialization</dt>
+<dd>The process of converting a block's attributes object into HTML markup, typically occurring when saving the post.</dd>
```

Comment:
Although I suppose it might be possible to override, my understanding is that the block is serialised to HTML every time a block is edited, not just when a post is saved.

Reply:
Yes, I think you are right. I updated to say the following which should cover it:

> The process of converting a block's attributes object into HTML markup, this occurs each time a block is edited.